dockerfiles/itest/itest/itest.py

```diff
@@ -132,15 +132,19 @@ def test_nerve_service_config(setup):
                 "type": "http",
                 "uri": "/http/service_three.main/1024/status",
                 "headers": {
-                    "Host": "www.test.com"
+                    "Host": "www.test.com",
                 },
-            }
+            },
         ],
         "host": MY_IP_ADDRESS,
         "port": 1024,
         "weight": CPUS,
         "zk_hosts": [ZOOKEEPER_CONNECT_STRING],
-        "zk_path": "/nerve/region:sjc-dev/service_three.main"
+        "zk_path": "/nerve/region:sjc-dev/service_three.main",
+        'labels': {
+            'weight': CPUS,
+            'region': 'sjc-dev',
```

Comment:
Is your intent with this one just to "test the waters" and eventually go wild and fill up this metadata?